cartographer_ros/launch/offline_backpack_2d_no_rviz.launch

```diff
@@ -0,0 +1,28 @@
+<!--
```

Comment:
Instead of having separate launcher for without rviz, why don't you add arguments 
`use_rviz` argument in offline_backpack_2d.launch.
```
<arg name="use_rviz" default="true">

  <node if="$(arg use_rviz)" name="rviz" pkg="rviz" type="rviz" required="true"
      args="-d $(find cartographer_ros)/configuration_files/demo_2d.rviz" />
   ...
```

Or 
```
<arg name="use_rviz" default="true">

<group if="$(arg use_rviz)">
   <node rviz>
   <node occupancy_grid>
</group>

```

Reply:
Awesome I did not know about this magic. Thanks.